packages/core/src/actions/invoke.ts

```diff
@@ -123,6 +144,7 @@ export function invoke<
   invoke.systemId = systemId;
   invoke.src = src;
   invoke.input = input;
+  invoke.subscribe = !!onSnapshot;
```

Comment:
I like this one very much - but perhaps we could rename this somehow? `syncSnapshot`? `iLikeChatter`? `listenToSnapshots`?

Reply:
`syncSnapshot` it is